TACC/Core-Portal
# task/FP-1633: Alter webhook and job name to use new tap functions

## Overview
VNC sessions now rely on a set of shared functions available on HPC systems called `TAP Functions`. In order to invoke the functions, we must have the substring `tap_` somewhere present in the job name. The session host is also changed from `vis.tacc` to `tap.tacc`.


## Related

* [FP-1633](https://jira.tacc.utexas.edu/browse/FP-1633)

## Testing

0. Ensure you have [ngrok running locally](https://github.com/TACC/Core-Portal#portal)
1. Find the private app `Paraview VNC` in My Apps
2. Run the app, and confirm you get a VNC session link, and the session works as expected
